advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.    nSeconds = nDays * 86400 + timestep*5      
</pre><p>should we generally split inputs that the user will change (nx, n_steps, configuration type etc) from diagnostics (very_n_steps, write initial etc) from derived but constant operations to make it easier?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>